esp/platform/application_config.cpp

```diff
@@ -436,6 +437,18 @@ void setLDAPSecurityInWSAccess(IPropertyTree *legacyEsp, IPropertyTree *legacyLd
     }
 }
 
+// Copy trace flags from appEsp to legacyEsp. The source is expected to be an application's `esp`
+// configuration object. The destination is expected to be the `EspProcess` element.
+void copyTraceFlags(IPropertyTree *legacyEsp, IPropertyTree *appEsp)
+{
+    IPropertyTree *traceFlags = appEsp->queryPropTree(propTraceFlags);
+    if (traceFlags)
```

Comment:
WIll do.

Reply:
The simplification of this method is straightforward. Adding a function to jptree ks less so. This function, making assumptions about source and target nodes, is too specialized to belong in jlib. My first thought for generalizing it involves target node, source node, and source path, with the copied node always added as a child of the target. This might not be general enough, if someone might want to specify a target path as well.

If a generalized copy function should be part of this PR, what would your expectation be for its interface/behavior?

Reply:
There is an existing copyTree function that is clone in workunit.cpp and espp.cpp.  It would make sense for this to be migrated into jptree, and for this function to make use of it.  Not for this PR though.